scaleph-ui-react/src/services/auth.ts

```diff
@@ -42,7 +42,7 @@ export async function isUserLoggedIn() {
 }
 
 export function hasPrivilege(code: string) {
-  let pCodes: string[] = JSON.parse(localStorage.getItem(USER_AUTH.pCodes) || '');
+  let pCodes: string[] = JSON.parse(localStorage.getItem(USER_AUTH.pCodes) || 'null');
```

Comment:
Please modify it to the following code
`  let pCodes: string[] = JSON.parse(localStorage.getItem(USER_AUTH.pCodes) + '');`